roles/openshift_facts/library/openshift_facts.py

```diff
@@ -56,6 +56,13 @@ def migrate_docker_facts(facts):
     if 'node' in facts and 'portal_net' in facts['node']:
         facts['docker']['hosted_registry_insecure'] = True
         facts['docker']['hosted_registry_network'] = facts['node'].pop('portal_net')
+
+    # log_options was originally meant to be a comma separated string, but
+    # we now prefer an actual list, with backward compatability:
+    if 'log_options' in facts['docker'] and \
+            isinstance(facts['docker']['log_options'], basestring):
+        facts['docker']['log_options'] = facts['docker']['log_options'].split(",")
```

Comment:
First pass through using a comma separated string, no openshift.fact cache in place, it does seem to be coming out ok I think: "log_options": ["max-size=1M", "max-file=3"]

Let me know if I've got anything mixed up here but I think it's getting picked up via init_local_facts calling the migration code.